### DISCUSSION TITLE
make a release?

### DISCUSSION THREAD

**Discussion by ossilator** [larsimmisch/pyalsaaudio#150] [General]
with a bunch of new features and bugfixes implemented, no outstanding high-priority issues, and nothing new expected to arrive anytime soon (i'm assuming that the loopback work is on the back-burner), i would suggest making a 0.11.0 release.

doc/README.md contains a release checklist.
it doesn't mention the need to update the CHANGES.md file, though.
anything else?

any takers? :wink:

**Comment by larsimmisch** [comment]
I think it's a good idea, and I might even get to it this week.

The loopback code is tricky. I still don't have it in a state where it is robust against XRUNs. It's also a bit more complicated than I'd like
